### ISSUE TITLE
A Kafka output for the alerter

### ISSUE THREAD

**Issue by rixed** [rixed/ramen#1049]
Ie. `ViaKafka of { brokers : string ; topic : string ; partition : int ; options : string list }` or similar.

The main concern is: what message format shall be used?

Option1:
Just turn all parameters into a giant json and send this;
- but what if we want something else than json?
- Also not convenient is the same alert is supposed to be sent via several channels with different formats

Option2:
Have a text template and send that.
- would work to send some json for sure, esp. if we had a template function to do proper formatting, but what about binary format such as avro?

Option3:
Let ramen do it with its own serializers
In other words, we are back at the original "notify" that could perform various tasks, with some ad-hoc generated code if needed - well, but it has been disposed of because workers needed to be free from such dependencies (esp we wanted a single point of connection rather than many movable ones).
Kafka may be a special case as we'd like to have an event exporter from the workers as well (as opposed to only a notification exporter).
